config/deploy/kubernetes/kubernetes-all.yaml

```diff
@@ -3229,6 +3229,255 @@ metadata:
 
 #     http://www.apache.org/licenses/LICENSE-2.0
 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dynatrace-kubernetes-monitoring
```

Comment:
why is this present multiple times ?

Reply:
it's generated from my untracked git leftovers:
`# Source: dynatrace-operator/templates/Common/kubernetes-monitoring/clusterrole-kubernetes-monitoring_BACKUP_1250775.yaml`
just another reason to make it autogenerated during build process and publish as build artifacts ;)